app/src/main/java/com/sunasterisk/a14day_challenge/data/local/DataBaseHandler.kt

```diff
@@ -143,6 +142,7 @@ class DataBaseHandler private constructor(context: Context) :
         if (user == null) {
             return null
         }
```

Comment:
Đoạn này phải là return false chứ  nhỉ 
```suggestion
    user ?: return false 
```